Content.Server/StationEvents/Events/BureaucraticErrorRule.cs

```diff
@@ -14,6 +14,9 @@ public sealed class BureaucraticErrorRule : StationEventSystem<BureaucraticError
 {
     [Dependency] private readonly StationJobsSystem _stationJobs = default!;
 
+    [ValidatePrototypeId<JobPrototype>]
+    public const string IgnoreRuleJob = "StationAi";
```

Comment:
Make this a hashset so you can exclude multiple jobs and move it to BureaucraticErrorRuleComponent so it can be set in yaml instead of being hardcoded.